service/src/main/kotlin/fi/espoo/evaka/EvakaEnv.kt

```diff
@@ -98,7 +99,8 @@ data class EvakaEnv(
                 feeDecisionGeneratorV2Enabled =
                     env.lookup("evaka.fee_decision.generator.v2_enabled") ?: false,
                 voucherValueDecisionGeneratorV2Enabled =
-                    env.lookup("evaka.voucher_value_decision.generator.v2_enabled") ?: false
+                    env.lookup("evaka.voucher_value_decision.generator.v2_enabled") ?: false,
+                titaniaEnabled = env.lookup("evaka.titania_enabled") ?: false
```

Comment:
This should probably be removed since it doesn't seem to be now used anywhere (?)

Reply:
Good catch, fixed.